internal/host/monitor.go

```diff
@@ -14,13 +14,6 @@ import (
 	"gorm.io/gorm"
 )
 
-func (m *Manager) SkipMonitoring(h *models.Host) bool {
-	skipMonitoringStates := []string{string(models.LogsStateCompleted), string(models.LogsStateTimeout), ""}
-	result := ((swag.StringValue(h.Status) == models.HostStatusError || swag.StringValue(h.Status) == models.HostStatusCancelled) &&
-		funk.Contains(skipMonitoringStates, h.LogsInfo))
-	return result
-}
-
 func (m *Manager) initMonitoringQueryGenerator() {
```

Comment:
Looks great , do we have a unit-test for it? if yes lets push

Reply:
yeah, noticed that yestarday that there were not unit tests, I added some!